docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -41,14 +41,16 @@ To isolate a host from a case alert:
 
 To isolate a host from an endpoint:
 
-. Go to *Manage -> Endpoints*, then select the appropriate endpoint in the *Endpoint* column. The endpoint details flyout opens.
-. Click *Take action -> Isolate host*.
+. Go to *Manage -> Endpoints*, then either:
+    * Select the appropriate endpoint in the *Endpoint* column, then click *Take action -> Isolate host* in the endpoint details flyout.
+    * Click the *Actions* menu (*...*) on the appropriate endpoint, then select *Isolate host*.
 . Enter a comment describing why you’re isolating the host (optional).
 . Click *Confirm*.
 
-After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*.
+After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*. Any user currently on the host receives a notification that the computer has been isolated from the network:
```

Comment:
```suggestion
After the host is successfully isolated, the *Agent Status* changes to *Isolated*. Any user currently on the host receives a notification that the computer has been isolated from the network:
```
Seems redundant to say that a status is added to agent status, also if you're looking on the agent table then Isolated appears under Agent status, not next to it.

Reply:
```suggestion
After the host is successfully isolated, an *Isolated* status is added to the endpoint. Any user currently on the host receives a notification that the computer has been isolated from the network:
```